src/pages/App.tsx

```diff
@@ -84,46 +98,51 @@ export default function App() {
             <Popups />
             <Polling />
             <TopLevelModals />
-            <Switch>
-              <Route exact strict path="/vote" component={Vote} />
-              <Route exact strict path="/vote/:governorIndex/:id" component={VotePage} />
-              <Route exact strict path="/claim" component={OpenClaimAddressModalAndRedirectToSwap} />
-              <Route exact strict path="/uni" component={Earn} />
-              <Route exact strict path="/uni/:currencyIdA/:currencyIdB" component={Manage} />
-
-              <Route exact strict path="/send" component={RedirectPathToSwapOnly} />
-              <Route exact strict path="/swap/:outputCurrency" component={RedirectToSwap} />
-              <Route exact strict path="/swap" component={Swap} />
-
-              <Route exact strict path="/pool/v2/find" component={PoolFinder} />
-              <Route exact strict path="/pool/v2" component={PoolV2} />
-              <Route exact strict path="/pool" component={Pool} />
-              <Route exact strict path="/pool/:tokenId" component={PositionPage} />
+            <Suspense fallback={<div>loading...</div>}>
```

Comment:
English text should be wrapped in `<Trans>` to get translations.

Use `<Dots>` in `components/swap/styleds` instead of `...`

Reply:
@callil did you ever look at what the suspense animation would look like? I like the running horse from v1..

Reply:
@justindomingue I've updated the pull request per your review, thanks for the guides.